README.md

```diff
@@ -10,33 +10,28 @@ It will also set up:
 * Automatic versioning using [bump2version](https://github.com/c4urself/bump2version)
```

Comment:
Now that we abandoned bump2version, we should mention setuptools_scm instead. Maybe `handling versions with setuptools_scm`